internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -760,6 +760,19 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 									actionParams.(map[string]interface{})[logCustomFields] = newLogCustomFields
 								}
 							}
+							// check if our ruleset is of action 'skip'
+							if rules.([]interface{})[ruleCounter].(map[string]interface{})["action"] == "skip" {
```

Comment:
is this specific to skip rules? can other actions do the same?

Reply:
I think this is the first time we are seeing something like this. Provider docs don't mention this would be specific to this action, so I'd assume other actions might as well use that. I'm a little afraid to open this to all actions right now as I don't know if it could break something. @jafowler what do you think this?

Reply:
So far its only applicable to the "skip" action. Thats not to say that there won't be more, but i think we could handle that on a case by case basis.